opyn/opyn/config.py

```diff
@@ -14,7 +14,10 @@ class AuthorizationPages:
 class OpynSDKConfig(SDKConfig):
 
     authorization_pages = AuthorizationPages
-    venue_chains = Chains
+    supported_chains = [
```

Comment:
So we are removing this from our internal code?

Reply:
Yes, exactly

Reply:
Nice!